src/Diagnostics/atmos_core.jl

```diff
@@ -0,0 +1,107 @@
+using ..Atmos
+using ..Atmos: thermo_state, turbulence_tensors
+using ..Mesh.Topologies
+using ..Mesh.Grids
+using ..MoistThermodynamics
+using LinearAlgebra
+
+function atmos_core_init(dgngrp::DiagnosticsGroup, currtime)
+    mpicomm = Settings.mpicomm
+    dg = Settings.dg
+    Q = Settings.Q
+    FT = eltype(Q)
+    grid = dg.grid
+    topology = grid.topology
+    N = polynomialorder(grid)
+    Nq = N + 1
+    Nqk = dimensionality(grid) == 2 ? 1 : Nq
+    nrealelem = length(topology.realelems)
+    nvertelem = topology.stacksize
+    nhorzelem = div(nrealelem, nvertelem)
+
+    if Array ∈ typeof(Q).parameters
+        localvgeo = grid.vgeo
+    else
+        localvgeo = Array(grid.vgeo)
+    end
+
+    @warn """
+Diagnostics ($dgngrp.name): not yet implemented"""
```

Comment:
Triple quotes don't need to begin at the beginning of the line.

```
julia>     """
           hello
           """
```
is the same as:
```
julia> """
       hello
       """